src/Form/Data/TableColumn.php

```diff
@@ -15,6 +15,8 @@ final class TableColumn
     private ?string $routeTarget = '_blank';
     private ?string $iconProperty = null;
     private ?bool $dateTimeProperty = null;
+    private ?bool $dataLinks = null;
+    private ?bool $arrayOfStrings = null;
```

Comment:
```suggestion
    private bool $noWrap = true;
```

Reply:
I fixed the above remarks in another branch (upload-file-view)